config/prow/build-cluster/create-build-cluster-helper.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
This should probably just be merged with the existing `create-build-cluster.sh` file.